ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-master.yaml

```diff
@@ -62,6 +62,12 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
+- as: e2e-component
+  commands: |
+    mkdir -p /tmp/artifacts
+    make test-e2e JUNITFILE=/tmp/artifacts/junit_report.xml --warn-undefined-variables
+  openshift_installer_src:
```

Comment:
Why not use the multi-stage workflow?

Reply:
https://docs.ci.openshift.org/docs/how-tos/migrating-template-jobs-to-multistage/#openshift_installer_src